package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "notificationapi-node-server-sdk",
-  "version": "0.4.0",
+  "version": "0.4.1",

```

Comment:
Please change to 0.5.0

Major: breaking changes
Minor: new features
Patch: bug fixes, no change in interfaces or intended behavior